### PR TITLE
Migrate more presubmits to build01: finished

### PR DESCRIPTION
**Pull request by hongkailiu**
The private repo has been handled by #12015.
So the target in #11906 for build01 has been achieved.

/cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

**Review by alvaroaleman — APPROVED** [review]

File: core-services/image-mirroring/redhat-developer/mapping_redhat-developer_latest_quay [hunk 1]
```diff
@@ -2,3 +2,10 @@ registry.svc.ci.openshift.org/redhat-developer/jenkins-operator:latest quay.io/r
 registry.svc.ci.openshift.org/redhat-developer/jenkins-operator:latest quay.io/redhat-developer/openshift-jenkins-operator:0.0.0
 registry.svc.ci.openshift.org/openshift/jenkins-operator:latest quay.io/redhat-developer/openshift-jenkins-operator:latest
 
+
+registry.svc.ci.openshift.org/redhat-developer/jenkins-operator-index:latest quay.io/redhat-developer/openshift-jenkins-operator-index:latest
+registry.svc.ci.openshift.org/redhat-developer/jenkins-operator-index:latest quay.io/redhat-developer/openshift-jenkins-operator-index:0.0.0
```
Comment (alvaroaleman) [inline core-services/image-mirroring/redhat-developer/mapping_redhat-developer_latest_quay:7]: /hold
what is this?

Comment (hongkailiu) [reply core-services/image-mirroring/redhat-developer/mapping_redhat-developer_latest_quay]: Not sure how it gets in.

```
git show --name-only | grep -v presubmits.yaml                                                                  
commit 06b7cceca065eee3e09fddb53db0ece11687cc62
Merge: e8922e8c14 61976bf416
Author: OpenShift Merge Robot <openshift-merge-robot@users.noreply.github.com>
Date:   Mon Sep 21 11:21:18 2020 -0400

    Migrate more presubmits to build01: finished

core-services/sanitize-prow-jobs/_config.yaml
```

**Review by alvaroaleman — APPROVED** [review]
/hold cancel

**Pull request merged**